lambico-datatest-jpa/src/main/java/org/lambico/datatest/junit/DatasetPath.java

```
@@ -0,0 +1,70 @@
+package org.lambico.datatest.junit;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.rules.ExternalResource;
+import org.lambico.datatest.json.DataAggregator;
+import org.reflections.Reflections;
+import org.reflections.scanners.ResourcesScanner;
+
+import java.io.*;
+import java.nio.charset.Charset;
+import java.util.*;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+/**
+ * Scan all the json file in a specific package.
+ */
+public class DatasetPath extends ExternalResource {
+    private String packageName;
+    private DataAggregator dataAggregator;
+
+    public DatasetPath(String packageName) {
+        this.packageName = packageName;
+    }
+
+    @Override
+    public void before() throws Throwable {
+        ObjectMapper mapper = new ObjectMapper();
+        mapper.disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
+        mapper.enable(DeserializationFeature.FAIL_ON_UNRESOLVED_OBJECT_IDS);
+        Reflections reflections = new Reflections(packageName, new ResourcesScanner());
+        Set<String> fileNames = reflections.getResources(Pattern.compile(".*\\.json"));
+        if(fileNames.isEmpty()){
```

I don't think it should throw an exception. An empty dataset could be a valid test case. Maybe we can substitute the throwing with a message in the log (warning or even info level).